BLSQ/iaso
# IA-2517: Invitation link leads to unusable page

Explain what problem this PR is resolving
- The expired or already used link lead to None page
- This ticket is about to display a message to let the user know that the link has been expired
Related JIRA tickets : [IA-2517](https://bluesquare.atlassian.net/browse/IA-2517)

## Self proofreading checklist


## Changes
- Made a check on the form if it's None
- If it's None display a message

## How to test
- Create a user and send a reset password through email
- Open the link and just modify on character in the token number and check the displayed message

## Print screen / video
![Screenshot from 2023-11-21 21-14-57](https://github.com/BLSQ/iaso/assets/19631540/c8a70650-9d03-4fdf-b004-fece286f0321)




[IA-2517]: https://bluesquare.atlassian.net/browse/IA-2517?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
